packages/logging-system/src/SetLocalStorage.ts

```diff
@@ -1,36 +1,32 @@
 import { LogRequestList, LogResponse, LogType } from './types/LogType';
 
 export const SetLocalStorageClear = () => {
-  const list: any[] = [];
+  const list: LogType[] = [];
   localStorage.setItem('yls-web', JSON.stringify(list));
 };
 
 export const SetLocalStorage = async (
   logger: LogType,
   putLog: (data: LogRequestList) => Promise<LogResponse>
 ) => {
-  if (window.localStorage.getItem('yls-web') == undefined) {
-    const list: any[] = [];
-    list.push(logger);
-    localStorage.setItem('yls-web', JSON.stringify(list));
-  } else {
-    const remainList: any[] = JSON.parse(localStorage.getItem('yls-web') as string) || [];
-    if (remainList.length < 10) {
-      const updateList = [...remainList, logger];
-      localStorage.setItem('yls-web', JSON.stringify(updateList));
-    } else {
-      const req: LogRequestList = {
-        logRequestList: remainList,
-      };
+  const list: LogType[] = JSON.parse(localStorage.getItem('yls-web') as string) || [];
+  list.push(logger);
+  localStorage.setItem('yls-web', JSON.stringify(list));
+
+  if (list.length >= 10) {
+    const req: LogRequestList = {
+      logRequestList: list,
+    };
+
+    SetLocalStorageClear();
 
-      try {
-        const res = await putLog(req);
-        if (res.success) {
-          SetLocalStorageClear();
-        }
-      } catch (e) {
-        console.error('Failed to post log');
-      }
+    try {
+      await putLog(req);
+    } catch (e) {
+      console.error('Failed to post log');
```

Comment:
이 오류는 프로덕션에는 안나면 좋겠네요. 어짜피 Network 탭에서 요청 상태를 확인 가능하니까요.

Reply:
혹시나 에러 로깅을 하고 싶으시다면 `e`도 포함해서 로깅해주시면 좋겠습니다.
```suggestion
      console.error('Failed to post log', e);
```